test/dialect/oracle/test_types.py

```diff
@@ -571,7 +572,7 @@ def _dont_test_numeric_nan_decimal(self, metadata, connection):
         )
 
     def test_numerics_broken_inspection(self, metadata, connection):
-        """Numeric scenarios where Oracle type info is 'broken',
+        """Numeric scenarios where Oracle Databasee type info is 'broken',
```

Comment:
**Michael Bayer** (zzzeek) wrote:

typo but dont worry about it

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5589